[2024-04-11T15:52:57.545+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-11T15:52:57.554+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_gg_bigquery_v07.create_dataset scheduled__2024-04-09T17:00:00+00:00 [queued]>
[2024-04-11T15:52:57.558+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_gg_bigquery_v07.create_dataset scheduled__2024-04-09T17:00:00+00:00 [queued]>
[2024-04-11T15:52:57.558+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 3
[2024-04-11T15:52:57.564+0000] {taskinstance.py:2327} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2024-04-09 17:00:00+00:00
[2024-04-11T15:52:57.570+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7761) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-11T15:52:57.571+0000] {standard_task_runner.py:63} INFO - Started process 7763 to run task
[2024-04-11T15:52:57.571+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_gg_bigquery_v07', 'create_dataset', 'scheduled__2024-04-09T17:00:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_gg_bigquery.py', '--cfg-path', '/tmp/tmpj5fbumi1']
[2024-04-11T15:52:57.573+0000] {standard_task_runner.py:91} INFO - Job 74: Subtask create_dataset
[2024-04-11T15:52:57.601+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_gg_bigquery_v07.create_dataset scheduled__2024-04-09T17:00:00+00:00 [running]> on host a4f2bd86ee13
[2024-04-11T15:52:57.658+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='duy' AIRFLOW_CTX_DAG_ID='dag_with_gg_bigquery_v07' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-09T17:00:00+00:00'
[2024-04-11T15:52:57.659+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-11T15:52:57.673+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-11T15:52:57.674+0000] {base.py:84} INFO - Using connection ID 'gcp' for task execution.
[2024-04-11T15:52:57.715+0000] {bigquery.py:470} INFO - datasetId was not specified in `dataset_reference`. Will use default value retail.
[2024-04-11T15:52:57.715+0000] {bigquery.py:470} INFO - projectId was not specified in `dataset_reference`. Will use default value ***-projecct.
[2024-04-11T15:52:57.716+0000] {bigquery.py:481} INFO - Creating dataset: retail in project: ***-projecct 
[2024-04-11T15:52:58.723+0000] {bigquery.py:2045} INFO - Dataset retail already exists.
[2024-04-11T15:52:58.758+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-11T15:52:58.770+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_gg_bigquery_v07, task_id=create_dataset, execution_date=20240409T170000, start_date=20240411T155257, end_date=20240411T155258
[2024-04-11T15:52:58.801+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-11T15:52:58.820+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296 AirflowProviderDeprecationWarning: Call to deprecated class BigQueryExecuteQueryOperator. (This operator is deprecated. Please use `BigQueryInsertJobOperator`.)
[2024-04-11T15:52:58.831+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-11T15:52:58.832+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
